src/plib/hooks/ContentInclude.php

```diff
@@ -67,20 +74,74 @@ public function addDnsSettingsMessages($idParam, $typeParam)
             list($type, $domainId) = explode(':', $idParam, 2);
 
             if (in_array($type, ['d', 's']) && is_numeric($domainId)) {
-                $domain = \pm_Domain::getByDomainId($domainId);
+                try {
+                    $domain = \pm_Domain::getByDomainId($domainId);
+                } catch (pm_Exception $e) {
+                    \pm_Log::warn($e);
+                }
 
                 $this->handleDomain($domain, true);
             }
         } elseif ($typeParam === 'domain') {
-            $domain = \pm_Domain::getByDomainId($idParam);
+            try {
+                $domain = \pm_Domain::getByDomainId($idParam);
+            } catch (pm_Exception $e) {
```

Comment:
because of this catch, you will have a fatal error below. why did you write it and did not even test?